doc: add depth param into concerned APIs doc part (#2810)
CanalTP/navitia: documentation/slate/source/includes/apis.md

-   Is there any perturbations on the "RER A" line ?
    -   <https://api.navitia.io/v1/coverage/fr-idf/networks/network:RER/lines/line:OIF:810:AOIF741/line_reports?>

Required | Name             | Type                            | Description                                         | Default Value
---------|------------------|---------------------------------|-----------------------------------------------------|--------------

> no       | since            | [iso-date-time](#iso-date-time) | Only display disruptions active after this date     |

Reviewer: ```suggestion
no       | since            | [iso-date-time](#iso-date-time) | Only display active disruptions after this date     |
```

> no       | until            | [iso-date-time](#iso-date-time) | Only display disruptions active before this date    |

Reviewer: ```suggestion
no       | until            | [iso-date-time](#iso-date-time) | Only display active disruptions before this date    |
```

no       | count            | int                             | Maximum number of results.                          | 10
no       | depth            | int                             | Json response [depth](#depth)                       | 1
no       | forbidden_uris[] | id                              | If you want to avoid lines, modes, networks, etc.   |
no       | disable_geojson  | boolean                         | remove geojson fields from the response             | false

> no       | tags[]           | array of string                 | Restrain the search within the given disruption tags|

Reviewer: ```suggestion
no       | tags[]           | array of string                 | Filter disruptions with the given tags|
```
